src/styles/utils/mixins.scss

```diff
@@ -1,11 +1,13 @@
-@mixin bigcircle {
-  width: $big-circle-size;
-  height: $big-circle-size;
-  border-radius: 50%;
+@mixin square ($size){
+  width: $size;
+  height: $size;
 }
 
-@mixin smallcircle {
-  width: $small-circle-size;
-  height: $small-circle-size;
+@mixin bigcircle ($size) {
+  @include square ($size);
   border-radius: 50%;
 }
+
+@mixin smallcircle ($size) {
+  @include bigcircle ($size * 0.05);
+}
```

Comment:
Please combine the mixins into one and specify the size difference when passing it to the new mixin.